inception-imls-external/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/external/ExternalRecommender.java

```diff
@@ -190,9 +191,11 @@ private String serializeTypeSystem(CAS aCas) throws RecommendationException
 
     private String serializeCas(CAS aCas) throws RecommendationException
     {
-        try (ByteArrayOutputStream out = new ByteArrayOutputStream()) {
-            XmiCasSerializer.serialize(aCas, null, out, true, null);
-            return new String(out.toByteArray(), UTF_8);
+        try (StringWriter out = new StringWriter()) {
+            XmiCasSerializer xmiCasSerializer = new XmiCasSerializer(null);
+            XMLSerializer sax2xml = new XMLSerializer(out, true);
+            xmiCasSerializer.serialize(aCas, sax2xml.getContentHandler(), null, null, null);
```

Comment:
Why do we needed to change this?

Reply:
Need is relative. By avoiding the raw byte representation, we can (try to) avoid the encoding issue altogether.